### PR TITLE
Show alert dialog card for sharing personal detail submission via email when clicking on the feedback menu.

### LINKED ISSUE #4732: Users sending feedback may not realize that it reveals the identity behind their wiki username
Some Mediawiki contributors would be at risk if their identity is revealed.
Problem: When **WhistleblowerFromNorthKorea123** sends feedback about the app, the following is posted to the public mailing list:

>'**Kim Jong-un**' via commons-app-android
>-- Technical information --
>API level: 30
>Android version: 11
>Device manufacturer: samsung
>Device model: SM-G986B
>Device: y2s
>Network type: wifi
>App version name: 3.1.1~1c9267ca0
>User name: **WhistleblowerFromNorthKorea123**
>Hi there is a typo in your SPARQL query.

Solution: I think we should **show a toast** like this when tapping "Feedback", while the email compose activity appears:

```
Please remove from this email any information that you are not comfortable sharing publicly.
```

_(I just changed the group's settings so that email addresses are not visible anymore, please let me know if they are still visible to non-managers)_

### PR DESCRIPTION
**Pull request by ratneshnavlakhe**
Show alert dialog card for sharing personal detail submission via email when clicking on the feedback menu.

**Description (required)**

Fixes #4732 

What changes did you make and why?

Add alert dialog card for sharing the personal detail submission via email client when clicking on feedback menu.

**Screenshots (for UI changes only)**

alert

### CODE REVIEW

**Comment by ratneshnavlakhe** [comment]
Hi @nicolas-raoul please review/approve this PR for running workflow, this needs approval. Let me know if I am missing anything here.

File: app/src/main/java/fr/free/nrw/commons/navtab/MoreBottomSheetLoggedOutFragment.java [hunk 1]
```diff
@@ -64,6 +65,20 @@ public void onLogoutClicked() {
...
+        new AlertDialog.Builder(getActivity())
+            .setMessage(R.string.feedback_sharing_data_alert)
+            .setCancelable(false)
+            .setPositiveButton(R.string.ok, (dialog, which) -> {
+                sendFeedback();
+            })
+            .show();
+    }
+
+    private void sendFeedback() {
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/navtab/MoreBottomSheetLoggedOutFragment.java:81]: javadoc

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -682,4 +682,5 @@ Upload your first media by tapping on the add button.</string>
   <string name="apply">Apply</string>
   <string name="reset">Reset</string>
   <string name="location_message">Location data helps Wiki editors find your picture, making it much more useful.\nYour recent uploads have no location.\nWe suggest you turn on location in your camera app\'s settings.\nThank you for uploading!</string>
+  <string name="feedback_sharing_data_alert">Please remove from this email any information that you are not comfortable sharing publicly</string>
```
Comment (nicolas-raoul) [inline app/src/main/res/values/strings.xml:685]: Sorry could you please change this to:

Please remove from this email any information that you are not comfortable sharing publicly. Also, please be aware the your email address with which you are posting, and the associated name and profile picture, will be visible publicly.

**Review by nicolas-raoul — CHANGES_REQUESTED** [review]

**Review by nicolas-raoul — CHANGES_REQUESTED** [review]
I just tested, the popup UI is perfect.
However, it does not show up when logged in.

File: app/src/main/java/fr/free/nrw/commons/navtab/MoreBottomSheetLoggedOutFragment.java [hunk 3]
```diff
@@ -40,7 +40,8 @@
     public View onCreateView(@NonNull final LayoutInflater inflater,
         @Nullable final ViewGroup container, @Nullable final Bundle savedInstanceState) {
         super.onCreateView(inflater, container, savedInstanceState);
-        final View view = inflater.inflate(R.layout.fragment_more_bottom_sheet_logged_out, container, false);
+        final View view = inflater
+            .inflate(R.layout.fragment_more_bottom_sheet_logged_out, container, false);
```
[the remark this answers is not in the document]
Comment (ratneshnavlakhe) [reply app/src/main/java/fr/free/nrw/commons/navtab/MoreBottomSheetLoggedOutFragment.java:44]: sure

File: app/src/main/res/values/strings.xml (hunk 2 above)
Comment (ratneshnavlakhe) [reply app/src/main/res/values/strings.xml:685]: updated!

**Comment by ratneshnavlakhe** [comment]
> I just tested, the popup UI is perfect. However, it does not show up when logged in.

I think I missed adding the similar case in Fragment named `MoreBottomSheetFragment.java` this handles the logged-in use case. Will update it.

**Comment by ratneshnavlakhe** [comment]
@nicolas-raoul have pushed the suggested changes, please take a look. Thanks

**Review by nicolas-raoul — APPROVED** [review]
Great javadoc, thanks!
The code looks fine too.
Tested, works perfectly.

**Pull request merged**